src/main/scala/uk/gov/nationalarchives/notifications/decoders/TransformEngineV2Decoder.scala

```diff
@@ -1,6 +1,8 @@
 package uk.gov.nationalarchives.notifications.decoders
```

Comment:
Can you rename the object to `ExportNotificationDecoder` to make it more generic. These message might be used by more services than just TRE